src/cid.rs

```diff
@@ -108,6 +108,19 @@ impl<const S: usize> Cid<S> {
         }
     }
 
+    /// Convert a CIDv0 to a CIDv1
```

Comment:
Nitpick (change will be approved and merge even without changing it): Perhaps adding information that if it is already a CIDv1, that it will be unchanged.

Reply:
I literally wrote it out, thought it was a bit verbose compared to the others and dropped it. Also test incoming :)